core-web/libs/edit-content/jest.config.ts

```diff
@@ -18,5 +18,6 @@ export default {
         'jest-preset-angular/build/serializers/no-ng-attributes',
         'jest-preset-angular/build/serializers/ng-snapshot',
         'jest-preset-angular/build/serializers/html-comment'
-    ]
+    ],
+    testEnvironment: '@happy-dom/jest-environment'
```

Comment:
What does this does?

Reply:
There is an [issue](https://github.com/primefaces/primeng/issues/14085) with Jest and PrimeNG when using the `@layer` CSS feature. The root cause of this issue is that JSDOM does not support this property. A possible solution is to use the `@happy-dom/jest-environment`, which provides support for the `@layer` CSS feature.